app/src/main/res/values/dimens.xml

```diff
@@ -31,10 +31,10 @@
     <dimen name="spacing_xxxlarge">96dp</dimen>
 
     <dimen name="spacing_tiny">2dp</dimen>
-
-    <dimen name="toolbar_elevation">12dp</dimen>
+    
     <dimen name="elevation_big">20dp</dimen>
     <dimen name="drawer_width">320dp</dimen>
+    <dimen name="default_icon_size">48dp</dimen>
 
     <dimen name="padding_normal">20dp</dimen>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/a9a8b9866fd50705acb7680bd2dc6bdf6144f0e8/app/src/main/res/values/dimens.xml#L39"></span>The resource <code>R.dimen.padding_normal</code> appears to be unused</td>
    </tr>
  </tbody>
</table>